modules/seqexec/model/src/main/scala/seqexec/model/package.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec
+
+import cats._
+import seqexec.model.enum._
+
+package object model {
+  type ParamName = String
+  type ParamValue = String
+  type Parameters = Map[ParamName, ParamValue]
+  type StepConfig = Map[SystemName, Parameters]
+  implicit val stEq: Eq[StepConfig] = Eq.fromUniversalEquals
+  type StepId = Int
```

Comment:
Would you switch this for `Index`?

Reply:
`Index` is a positive short, so yeah anything where you start numbering at 1.